### DISCUSSION TITLE
Voice notes

### DISCUSSION THREAD

**Discussion by greenfoo** [organicmaps/organicmaps#1932] [Ideas]
While driving, sometimes I see something that needs to be fixed (for example, when the GPS tries to get me into a street in the wrong direction).

It would be nice if I could do something "very easy" (so that I don't take my eyes off the road) to take note of this.

I was thinking in double tapping the screen and then, while the finger stays in the screen, proceed to record a voice message which would be saved (together with the current GPS coordinates) for later.

This could be used also for taking OSM notes while walking, hiking, etc... when you don't want to stop and begin typing on the keyboard for whatever reason.

Some more ideas: the recording would be listed and the user would be able to listen and/or delete them from within the app. For extra points, the app would **transcribe** them to text and then upload them as OSM notes to the user profile!

**Comment by biodranik** [comment]
I am also often in a similar situation. It's hard to do any touches on the road though. On iOS with the Internet connection and CarPlay, there is Siri who may help to do something. One of the popular cases is to add a stationary camera, for example.

It could be possible to display a "record" button In the app's navigation mode, but then the recording should be stopped by another touch.

On the walk, it could be easier and faster to edit the map directly using some other map editors.
